functions/index.js

```diff
@@ -1,9 +1,65 @@
 const functions = require('firebase-functions')
+const axios = require('axios')
+const mailchimp = require('@mailchimp/mailchimp_marketing')
 
-// Create and Deploy Your First Cloud Functions
-// https://firebase.google.com/docs/functions/write-firebase-functions
+exports.processCalendlyEventInvitee = functions.https.onRequest(async (request, resp) => {
+  const { calendlyInviteeURI } = request.body
+  const token = 'eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL2F1dGguY2FsZW5kbHkuY29tIiwiaWF0IjoxNjE4NjA3MDM3LCJqdGkiOiIxZjliMjRjOC03NGEwLTRlNzMtOGMxMy0xMGI0MjNlYmI2NjQiLCJ1c2VyX3V1aWQiOiJHR0hBUVVBQzNONUxUUlgyIiwib3JnX3V1aWQiOiJESENIVVJEQTZZWEJRUzZMIn0.UxvkeOZGkCiq_9RE6bU2PV-mD0keGKE7rwuvOZ2SZLY'
+  const listId = 'bfb4a12429'
```

Comment:
Move these variables to the outside scope (line 4). These variables are constants, and never change for the whole execution of your function. As such, they need to be defined on the outside scope and their names must be in uppercase: `CALENDLY_ACCESS_TOKEN` and `MAILCHIMP_LIST_ID`.